controllers/license/license.js

```diff
@@ -360,86 +380,105 @@ function sendResponse(passon) {
                 }
             }
             else if(oddata === 'unknown') {
                 if(osicode === true && odtext === true) {
-                    passon.res.redirect('https://img.shields.io/badge/licence-mostly%20open-yellow.svg');
+                    badgeString = 'licence-mostly%20open-yellow.svg';
                 }
                 else if(osicode === true && odtext === false) {
-                    passon.res.redirect('https://img.shields.io/badge/licence-partially%20open-fe7d00.svg');
+                    badgeString = 'licence-partially%20open-fe7d00.svg';
                 }
                 else if(osicode === false && odtext === true) {
-                    passon.res.redirect('https://img.shields.io/badge/licence-partially%20open-fe7d00.svg');
+                    badgeString = 'licence-partially%20open-fe7d00.svg';
                 }
                 else if(osicode === false && odtext === false) {
-                    passon.res.redirect('https://img.shields.io/badge/licence-closed-ff0000.svg');
+                    badgeString = 'licence-closed-ff0000.svg';
                 }
                 else if(osicode === false && odtext === 'unknown') {
-                    passon.res.redirect('https://img.shields.io/badge/licence-closed-ff0000.svg');
+                    badgeString = 'licence-closed-ff0000.svg';
                 }
                 else if(osicode === true && odtext === 'unknown') {
-                    passon.res.redirect('https://img.shields.io/badge/licence-partially%20open-fe7d00.svg');
+                    badgeString = 'licence-partially%20open-fe7d00.svg';
                 }
             }
             else if(odtext === 'unknown') {
                 if(osicode === true && oddata === true) {
-                    passon.res.redirect('https://img.shields.io/badge/licence-mostly%20open-yellow.svg');
+                    badgeString = 'licence-mostly%20open-yellow.svg';
                 }
                 else if(osicode === true && oddata === false) {
-                    passon.res.redirect('https://img.shields.io/badge/licence-partially%20open-fe7d00.svg');
+                    badgeString = 'licence-partially%20open-fe7d00.svg';
                 }
                 else if(osicode === false && oddata === true) {
-                    passon.res.redirect('https://img.shields.io/badge/licence-partially%20open-fe7d00.svg');
+                    badgeString = 'licence-partially%20open-fe7d00.svg';
                 }
                 else if(osicode === false && oddata === false) {
-                    passon.res.redirect('https://img.shields.io/badge/licence-closed-ff0000.svg');
+                    badgeString = 'licence-closed-ff0000.svg';
                 }
             }
+
+            passon.res.tracking.service = passon.service;
+            let redirectURL = config.badge.baseURL + badgeString + config.badge.options;
+            passon.res.redirect(redirectURL);
         }
         fulfill(passon);
```

Comment:
It might be hard to catch errors here, suggest to move `fulfill(passon)` directly after the call to `passon.res.redirect(..)` (just like in other functions in this file).